SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/shared/JsonObjectFileHelper.java

```diff
@@ -28,17 +29,21 @@ public JsonObjectFileHelper(ObjectMapper objectMapper) {
 
     log.info("Reading {}", file);
 
-    byte[] bytes = Files.readAllBytes(file);
+    byte[] bytes = new byte[0];
```

Comment:
nitpick -- i dont think we need to bother giving this a default, we could just move the variable into the try-catch